x/evm/types/utils.go

```diff
@@ -165,3 +166,22 @@ func recoverEthSig(R, S, Vb *big.Int, sigHash ethcmn.Hash) (ethcmn.Address, erro
 
 	return addr, nil
 }
+
+// HexToBytes turns a 0x prefixed hex string into a byte slice
+func HexToBytes(in string) ([]byte, error) {
```

Comment:
isn't there a util on ethereum for this function?

Reply:
yeah probably, I won't need it once I switch to ethcmn.Hex